doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -78,6 +78,7 @@ Thread
 * Memory requirements page shows requirements based on the configuration used for certification instead of minimal configuration which has been removed.
 * Updated "Configuring a radio co-processor" section on Thread Border Router page with the information about forcing Hardware Flow Control in JLink.
 * Updated nrfconnect/otbr docker.
+* Updated OpenThread pre-build libraries for certification.
```

Comment:
```suggestion
* Updated OpenThread pre-build libraries.
```
Libraries can be used by customers for any purpose.

Also I think it's important to state that Thread 1.1 libraries are being removed.